<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="triggerWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>58</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="sampleRadio">
          <property name="statusTip">
           <string>Sample Rate-Triggered: Script is executed at a defined sample rate</string>
          </property>
          <property name="text">
           <string>Samplerate-Triggered</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="signalRadio">
          <property name="statusTip">
           <string>Signal-Triggered: Script is linked to device sample rate and executed</string>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Signal-Triggered</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="triggerSamplerateSpinBox">
          <property name="suffix">
           <string> Hz</string>
          </property>
          <property name="maximum">
           <double>999999.989999999990687</double>
          </property>
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QListWidget" name="triggerSignals">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
